### ISSUE TITLE
Visual aid for CORSPEDIA using Japanese candlesticks technique

### ISSUE THREAD

**Issue by cuthbertyeo** [nusmodifications/nusmods#76]
The current corspedia provides 5 important variables from the CORS archive: 

1) Quota (Supply Volume available)
2) Bidders (Demand)
3) Lowest bid (lowest tip of candle wick)
4) Lowest succeeding bid (cutoff points / start of candle-stick body)
5) Highest bid (End of candle-stickbody / top of candle wick tip if highest bidder lowers his bet during closed bidding) 

![image](https://cloud.githubusercontent.com/assets/8245609/3672929/a9f460f2-1266-11e4-8881-0e73d81fcd90.png)

The string of these 5 important values merely arranged in an excel like format segregated based on faculty followed by its rounds lends no clue to students whom may be less sensitive to numbers. Students can better visualised when combining all the rounds for a particular module as shown below. 

![image](https://cloud.githubusercontent.com/assets/8245609/3672941/be7cadf4-1266-11e4-86ea-fa0b70144db8.png)

Key examples would include the popular "hammer" in any basic technical analysis book. This is define by a long bottom candle wick and a short candle stick body (which signifies that the lowest succeeding bid is very close range to highest bid mainly because students are holding on to their bets in open rounds and hence we would expect inflation only to occur in closed rounds. 

Another legit reason is that more people are willing to part away with more points and hence the close competition (small candlestick body). Hence, a higher probability in losing the bid for that module. 

In a nutshell, these values can be better comprehended by visual aids such as japanese candlesticks method which is popular amongst engineering and business students as it lends us a clearer picture behind those string of values more effectively.

**Comment by yangshun** [comment]
Thanks @cuthbertyeo but I think we agree that the current implementation serves the intended purpose. Hence, closing this (:
